jedi/common.py

```diff
@@ -155,6 +154,11 @@ def splitlines(string):
     return re.split('\n|\r\n', string)
 
 
+def content(file_path):
+    with open(file_path, 'r') as f:
+        return f.read()
```

Comment:
You shouldn't even be reading files. This will cause you issues with unicode. There's quite a bit of logic within Jedi that cares for this stuff. There's probably a code attribute in all the parsers that you can use.